src/client/renderingengine.cpp

```diff
@@ -557,34 +557,26 @@ void RenderingEngine::draw_scene(video::SColor skycolor, bool show_hud,
 
 const char *RenderingEngine::getVideoDriverName(irr::video::E_DRIVER_TYPE type)
 {
-	static const char *driver_ids[] = {
-			"null",
-			"software",
-			"burningsvideo",
-			"direct3d8",
-			"direct3d9",
-			"opengl",
-			"ogles1",
-			"ogles2",
+	static const std::unordered_map<irr::video::E_DRIVER_TYPE,const std::string> driver_ids = {
```

Comment:
canno we just have a global map having both this elemnt and the next string too, global one is better than 3 lists of elements refering to the sames, this clearly should be improved to increase maintenance

Reply:
#11413